blockchain/runtime/src/lib.rs

```diff
@@ -295,15 +301,16 @@ construct_runtime!(
         NodeBlock = opaque::Block,
         UncheckedExtrinsic = UncheckedExtrinsic
     {
-        System: frame_system::{Pallet, Call, Config, Storage, Event<T>},
-        RandomnessCollectiveFlip: pallet_randomness_collective_flip::{Pallet, Call, Storage},
-        Timestamp: pallet_timestamp::{Pallet, Call, Storage, Inherent},
-        Aura: pallet_aura::{Pallet, Config<T>},
-        Grandpa: pallet_grandpa::{Pallet, Call, Storage, Config, Event},
-        Balances: pallet_balances::{Pallet, Call, Storage, Config<T>, Event<T>},
-        TransactionPayment: pallet_transaction_payment::{Pallet, Storage},
-        Sudo: pallet_sudo::{Pallet, Call, Config<T>, Storage, Event<T>},
-        FractalMinting: fractal_minting::{Pallet, Call, Storage, Config<T>, Event<T>},
+        System: frame_system::{Pallet, Call, Config, Storage, Event<T>} = 0,
+        RandomnessCollectiveFlip: pallet_randomness_collective_flip::{Pallet, Call, Storage} = 1,
+        Timestamp: pallet_timestamp::{Pallet, Call, Storage, Inherent} = 2,
+        Aura: pallet_aura::{Pallet, Config<T>} = 3,
+        Grandpa: pallet_grandpa::{Pallet, Call, Storage, Config, Event} = 4,
+        Balances: pallet_balances::{Pallet, Call, Storage, Config<T>, Event<T>} = 5,
+        TransactionPayment: pallet_transaction_payment::{Pallet, Storage} = 6,
+        Sudo: pallet_sudo::{Pallet, Call, Config<T>, Storage, Event<T>} = 7,
+        FractalMinting: fractal_minting::{Pallet, Call, Storage, Config<T>, Event<T>} = 8,
+        Utility: pallet_utility::{Pallet, Call, Event} = 9,
```

Comment:
If you'd like, you can group these visually by their function. Just make sure to keep the indexes of existing pallets matching what they are now.